### PR TITLE
feat: Update SSO and Internal LB

### PR DESCRIPTION
**Pull request by jjrcluedin**
## Description
- Updates Load Balancer documentation to be clearer and more up to date
- Updates SSO documentation to be clearer and more up to date
Work Item ID: AB#

### CODE REVIEW

File: docs/020-deployment/howtos/020-setup-sso.md [hunk 1]
```diff
@@ -41,10 +41,11 @@ Registering your application establishes a trust relationship between your appli
 1. Under **Manage**, select **App registrations** > **New registration**.
 1. Enter a display **Name** for your application.
 ![Register_application_Name.png](../../assets/images/ama/howtos/configure-sso-2.png)
-1. Select the account types that can use the application.
+1. Select the **Supported account types** that can use the application.
+1. Leave Redirect URI empty for the time being.
 1. Select **Register**.
 
-    When the registration finishes, the Azure portal displays the **Overview** pane of the application registration. Here you can see the **Application (client) ID**. This value uniquely identifies your application. You'll need it to [enable SSO via Helm](#create-kubernetes-secret-and-enable-sso-via-helm).
+    When the registration finishes, the Azure portal displays the **Overview** pane of the application registration. Here you can see the **Application (client) ID**. This value uniquely identifies your application. Make note of this Id as you'll need it to [enable SSO via Helm](#create-kubernetes-secret-and-enable-sso-via-helm).
```
Comment (SolomiiaSnihur) [inline docs/020-deployment/howtos/020-setup-sso.md:48]:
```suggestion
    When the registration finishes, the Azure portal displays the **Overview** pane of the application registration. Here you can see the **Application (client) ID**. This value uniquely identifies your application. Make note of this ID as you'll need it to [enable SSO via Helm](#create-kubernetes-secret-and-enable-sso-via-helm).
```

File: docs/020-deployment/howtos/020-setup-sso.md [hunk 2]
```diff
@@ -137,13 +135,16 @@ You need to register a web API with the Microsoft identity platform and expose i
...
-2. Select **Expose an API**.
-3. In the **Scopes defined by this API** section, select **Add a scope**.
-4. Specify the following scope attributes:
-- **Scope name** – `https://www.cluedin.net/sso/user_impersonation`
-- **Who can consent** – **Admins and Users**
-- **Authorized client applications** – `https://www.cluedin.net/sso/user_impersonation`
-![expose_api.png](../../assets/images/ama/howtos/expose_api.png)
+1. Select **Expose an API**.
+1. In the **Scopes defined by this API** section, select **Add a scope**.
+1. For the first time setup, you will need to specify the `Application ID URI`. For this, use your frontend url with `/sso` appended. (ie. `https://cluedin.yourdomain.com/sso`)
```
Comment (SolomiiaSnihur) [inline docs/020-deployment/howtos/020-setup-sso.md:140]:
```suggestion
1. For the first-time setup, you will need to specify the `Application ID URI`. For this, use your front-end URL with `/sso` appended. (i.e. `https://cluedin.yourdomain.com/sso`)
```

File: docs/020-deployment/howtos/020-setup-sso.md [hunk 3]
```diff
@@ -137,13 +135,16 @@ You need to register a web API with the Microsoft identity platform and expose i
...
+    ![configure-sso-scopename.png](../../assets/images/ama/howtos/configure-sso-scopename.png)
+1. Specify the following scope attributes and then click `Add scope`:
```
Comment (SolomiiaSnihur) [inline docs/020-deployment/howtos/020-setup-sso.md:142]:
```suggestion
1. Specify the following scope attributes and then select `Add scope`:
```

File: docs/020-deployment/howtos/020-setup-sso.md [hunk 4]
```diff
@@ -41,10 +41,11 @@ Registering your application establishes a trust relationship between your appli
 1. Under **Manage**, select **App registrations** > **New registration**.
 1. Enter a display **Name** for your application.
 ![Register_application_Name.png](../../assets/images/ama/howtos/configure-sso-2.png)
-1. Select the account types that can use the application.
+1. Select the **Supported account types** that can use the application.
+1. Leave Redirect URI empty for the time being.
```
Comment (SolomiiaSnihur) [inline docs/020-deployment/howtos/020-setup-sso.md:45]:
```suggestion
1. Leave **Redirect URI** empty for the time being.
```

**Review by SolomiiaSnihur — CHANGES_REQUESTED** [review]

**Comment by jjrcluedin** [comment]
@SolomiiaSnihur There was quite a bit of merge conflicts, could I please just get another once over by yourself and make sure it's all good?

File: docs/kb/kb1001-202401upgrade.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ---
-layout: cluedin
+layout: default
```
Comment (SolomiiaSnihur) [inline docs/kb/kb1001-202401upgrade.md:2]:
```suggestion
layout: cluedin
```

Comment (SolomiiaSnihur) [reply docs/kb/kb1001-202401upgrade.md:2]: Layout should be "cluedin". This is the new look of the doc portal.

**Review by SolomiiaSnihur — CHANGES_REQUESTED** [review]
All good. Change layout to cluedin in the upgrade article.

**Pull request merged**